common/log/tag/tags.go

```diff
@@ -544,6 +544,21 @@ func ShardQueueAcks(categoryName string, ackLevel interface{}) ZapTag {
 
 // task queue processor
 
+// QueueReaderID returns tag for queue readerID
+func QueueReaderID(readerID int32) ZapTag {
+	return NewInt32("queue-reader-id", readerID)
+}
+
+// QueueAlertType returns tag for queue alert type
+func QueueAlertType(alertType string) ZapTag {
+	return NewStringTag("queue-alert-type", alertType)
+}
+
+// QueueAlertAttributes returns tag for queue alert attributes
+func QueueAlertAttributes(attributes interface{}) ZapTag {
+	return NewAnyTag("queue-alert-attributes", attributes)
```

Comment:
please try and see if the log is able to print readable things

Reply:
yes, verified. It will log something like for following
```
{"level":"info","ts":"2022-08-25T11:34:05.031-0700","msg":"Action completed for queue alert","queue-alert-type":"ReaderStuck","queue-alert-attributes":{"ReaderID":1,"CurrentWatermark":{"FireTime":"2055-02-16T08:22:57.924852959-08:00","TaskID":649244596167644460}},"logging-call-at":"mitigator.go:107"}
```